account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -0,0 +1,820 @@
+                elif not il_vals.get('name'):
+                    il_vals['name'] = _('MISSING DESCRIPTION')
+                if start_end_dates_installed:
+                    il_vals['start_date'] =\
+                        line.get('date_start') or parsed_inv.get('date_start')
+                    il_vals['end_date'] =\
+                        line.get('date_end') or parsed_inv.get('date_end')
+                uom = bdio._match_uom(
+                    line.get('uom'), parsed_inv['chatter_msg'])
+                il_vals['uom_id'] = uom.id
+                il_vals.update({
+                    'quantity': line['qty'],
+                    'price_unit': line['price_unit'],  # TODO fix for tax incl
+                })
+                vals['invoice_line_ids'].append((0, 0, il_vals))
+        # Write analytic account + fix syntax for taxes
+        aacount_id = config.get('account_analytic') and\
+            config['account_analytic'].id or False
+        if aacount_id:
+            for line in vals['invoice_line_ids']:
+                line[2]['account_analytic_id'] = aacount_id
+        return (vals, config)
+
+    @api.model
+    def set_1line_price_unit_and_quantity(self, il_vals, parsed_inv):
+        """For the moment, we only take into account the 'price_include'
+        option of the first tax"""
+        il_vals['quantity'] = 1
+        il_vals['price_unit'] = parsed_inv.get('amount_total')
+        if il_vals.get('invoice_line_tax_ids'):
+            for tax_entry in il_vals['invoice_line_tax_ids']:
+                if tax_entry:
+                    tax_id = False
+                    if tax_entry[0] == 4:
+                        tax_id = tax_entry[1]
+                    elif tax_entry[0] == 6:
+                        tax_id = tax_entry[2][0]
+                    if tax_id:
+                        first_tax = self.env['account.tax'].browse(tax_id)
+                        if not first_tax.price_include:
+                            il_vals['price_unit'] = parsed_inv.get(
+                                'amount_untaxed')
+                            break
+
+    @api.model
+    def set_1line_start_end_dates(self, il_vals, parsed_inv):
+        """Only useful if you have installed the module account_cutoff_prepaid
+        from https://github.com/OCA/account-closing"""
+        ailo = self.env['account.invoice.line']
+        if (
+                parsed_inv.get('date_start') and
+                parsed_inv.get('date_end') and
+                hasattr(ailo, 'start_date') and
+                hasattr(ailo, 'end_date')):
+            il_vals['start_date'] = parsed_inv.get('date_start')
+            il_vals['end_date'] = parsed_inv.get('date_end')
+
+    @api.multi
+    def parse_invoice(self):
+        self.ensure_one()
+        assert self.invoice_file, 'No invoice file'
+        logger.info('Starting to import invoice %s', self.invoice_filename)
+        file_data = self.invoice_file.decode('base64')
```

Comment:
Use `base64.b64decode(self.invoice_file)`

Reply:
I think there is something wrong with the importing anyway. I'm trying it but I get an error.

Reply:
see this: https://github.com/OCA/edi/pull/42#pullrequestreview-92928510